File: src/main/java/dev/fringe/crud/listener/EventListener.java
package dev.fringe.crud.listener;

import dev.fringe.crud.event.Event;
import org.springframework.context.ApplicationListener;
import org.springframework.stereotype.Component;

@Component
public   class EventListener implements ApplicationListener<Event> {

    public void onApplicationEvent(Event event) {
            System.out.println("Event : " + event);
    }
}

File: src/main/scala/dev/fringe/crud/listener/EventListener.scala
package dev.fringe.crud.listener;

import dev.fringe.crud.event.Event
import org.springframework.context.ApplicationListener
import org.springframework.stereotype.Component;

@Component class EventListener extends  ApplicationListener[Event[Object]] {
  override def onApplicationEvent(e: Event[Object]): Unit = System.out.println("Event : " + e)
}
